### ISSUE TITLE
Reference check does not support "must NOT exist"

### ISSUE THREAD

**Issue by nikolaPlume** [sodadata/soda-core#1816]
```
checks for dim_customers_dev:
  - values in (last_name, first_name) must exist in dim_customers_prod (last_name, first_name)
```

**`must exist`** is a really great thing, as part of **Reference checks**, however it would be good to have something like **`must NOT exist`**.

What would be the use case?
Some personal identifying information have to be de-identified and we need to verify that this is done properly. So, instead of checking that column values from one tables must exists in a second table's column, it would be useful to check totally opposite thing. To check that values (let's say email, phone_numbers) from original table, `must NOT exist` in a similar column in another external/de-identified table.

Although, there are many things you can add in your reference check, which could be more or less complex, adding something opposite to must exist, should not be a big deal.

**Comment by jmarien** [comment]
SODA-1455

**Comment by m1n0** [comment]
Support for `must not exist` has been released in [Soda Core 3.0.49](https://github.com/sodadata/soda-core/releases/tag/v3.0.49)
